assets/js/base/components/product-price/index.tsx

```diff
@@ -191,17 +191,17 @@ export interface ProductPriceProps {
 	/**
 	 * Currency configuration object
 	 */
-	currency: Currency | Record< string, never >;
+	currency?: Currency | Record< string, never >;
```

Comment:
This part had been changed to prevent a TS error in the following sections:
- https://github.com/woocommerce/woocommerce-blocks/blob/8e8b4a003d0d95c98b0cb388d034a0b73709f1e2/assets/js/atomic/blocks/product-elements/price/block.tsx#L46
- https://github.com/woocommerce/woocommerce-blocks/blob/8e8b4a003d0d95c98b0cb388d034a0b73709f1e2/assets/js/atomic/blocks/product-elements/price/block.tsx#L59-L76

Reply:
Making `currency` field optional will fix the TS errors in above files, but it will introduce more TS errors in `assets/js/base/components/product-price/index.tsx` file. For example, in following line:

https://github.com/woocommerce/woocommerce-blocks/blob/ae10c57e525e1d14b3f83689789765a298e66793/assets/js/base/components/product-price/index.tsx#L298

Therefore, now you will need to fix the TS errors related to making `currency` field optional 🙂

Reply:
Please ignore the above comment; after pulling the latest updates, it looks good 🤦‍♂️